test/parallel/test-crypto-ecdh-convert-key.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+const common = require('../common');
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const crypto = require('crypto');
+
+const { ECDH, getCurves } = crypto;
+
+// A valid private key for the secp256k1 curve.
+const cafebabeKey = 'cafebabe'.repeat(8);
+// Associated compressed and uncompressed public keys (points).
+const cafebabePubPtComp =
+    '03672a31bfc59d3f04548ec9b7daeeba2f61814e8ccc40448045007f5479f693a3';
+const cafebabePubPtUnComp =
+    '04672a31bfc59d3f04548ec9b7daeeba2f61814e8ccc40448045007f5479f693a3' +
+    '2e02c7f93d13dc2732b760ca377a5897b9dd41a1c1b29dc0442fdce6d0a04d1d';
+
+// Invalid test: key argument is undefined.
+common.expectsError(
+  () => ECDH.convertKey(),
+  {
+    type: TypeError,
+    message: 'Public key must be a buffer'
+  });
+
+// Invalid test: curve argument is undefined.
+common.expectsError(
+  () => ECDH.convertKey(cafebabePubPtComp),
+  {
+    type: TypeError,
+    message: 'ECDH curve name must be a string'
+  });
+
+// Invalid test: curve argument is invalid.
+common.expectsError(
+  () => ECDH.convertKey(cafebabePubPtComp, 'badcurve'),
+  {
+    type: TypeError,
+    message: 'Invalid ECDH curve name'
+  });
+
+const availableCurves = new Set(getCurves());
+
+if (availableCurves.has('secp256k1')) {
```

Comment:
Tiny nit: No need for a set, just use `if (getCurves().includes('secp256k1')) {`.

Reply:
Fixed! :)